app/Http/Controllers/ApiPhotosController.php

```diff
@@ -264,11 +244,13 @@ public function check ()
     {
         $user = Auth::guard('api')->user();
 
-        $photos = $user->photos()->where('verification', 0)->select('id', 'filename')->get();
-
-        if ($photos) return ['photos' => $photos];
+        $photos = $user->photos()
+            ->where('verified', 0)
+            ->where('verification', 0)
+            ->select('id', 'filename')
+            ->get();
 
-        return ['photos' => 'none'];
+        return ['photos' => $photos];
```

Comment:
Is this still used? Seems unusual to return an array of all photos. Could be large.

Reply:
We already return an array of all photos, but apparently, that has never been a problem since we return only one user's photos. This part of code that returns 'none' was never reached actually, the `if` condition was wrong. If there are no photos then the $photos array will be empty.

Reply:
```php
        $photos = $user->photos()->where('verification', 0)->select('id', 'filename')->get();

        if ($photos) return ['photos' => $photos];

        return ['photos' => 'none'];
```
This was the original condition. You'll notice `$photos` is a collection instance, which means even if it's empty, `if ($photos)` will always be true. This means we never return the 'none' string, if `$photos` is empty we simply return an array. My refactoring does not change anything, it simply removes the unreached code.